sdk/storage/azure-storage-files-datalake/src/datalake_directory_client.cpp

```diff
@@ -110,7 +110,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     protocolLayerOptions.SourceIfModifiedSince = options.SourceAccessConditions.IfModifiedSince;
     protocolLayerOptions.SourceIfUnmodifiedSince = options.SourceAccessConditions.IfUnmodifiedSince;
     protocolLayerOptions.RenameSource
-        = "/" + m_pathUrl.GetPath() + _internal::UrlEncodePath(fileName);
+        = "/" + m_pathUrl.GetPath() + "/" + _internal::UrlEncodePath(fileName);
```

Comment:
Do we have tests that would catch this? If not, add tests.

Reply:
there's a bug here. Working on it. will add tests